mediacenter/tests/test_search.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from search.models import Search
+
+from ..models import Document
+
+pytestmark = pytest.mark.django_db
+
+
+def test_document_are_indexed(video):
+    assert Search.objects.count() == 1
+    assert len(list(Search.search(public=True))) == 1
+    assert Document.objects.search("Ikinyugunyugu").count() == 0
```

Comment:
That's true, but, as someone said somewhere in this review, "explicit is better than implicit" ;)

What I mean is that I find both clearer and stronger to make sure the context is as expected before doing the actual "unit" test.

I'll not fight to convince that's a good practice, but I think when in doubt, better more tests than not enough :)